src/ensembl/src/shared/components/view-in-app/ViewInApp.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { mount } from 'enzyme';
+
+import ViewInApp from './ViewInApp';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+
+jest.mock('react-router-dom', () => ({
+  Link: () => <div>Link</div>
+}));
+
+describe('<ViewInApp />', () => {
+  it('contains Link and ImageButton', () => {
```

Comment:
The description says, "and ImageButton"; the body of the test does not check anything about the ImageButton.

But more to the point, what is the test actually trying to check? If I were writing the tests for this component, I would probably be interested in which buttons are rendered in response to which props, and what urls are passed to what buttons.

Reply:
Yes test needs improvement. Will ping you when I start working on this.